backend/src/main/java/pl/edu/agh/api/actors/Fighter.java

```diff
@@ -5,32 +5,19 @@
 import javax.persistence.Entity;
 import javax.persistence.Id;
 import javax.persistence.Table;
+import java.util.UUID;
 
 @Entity
 @Table
-@Getter
-@Setter
-@ToString
+@Data
```

Comment:
You should avoid using @Data with JPA Entites. See: https://www.jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/